### PR TITLE
feat: use `session.max_age` to set the pipeline auth token's expiration

### PR DESCRIPTION
**Pull request by rapsealk**
This is a follow-up for #1350 
This PR changes the JWT for the pipeline service to use `session.max_age` from the webserver.conf file to set its expiration.

**Checklist:** (if applicable)

- [x] Milestone metadata specifying the target backport version
- [x] Mention to the original issue
- [ ] Documentation
- [ ] API server-client counterparts (e.g., manager API -> client SDK)
- [ ] Test case(s) to demonstrate the difference of before/after

### CODE REVIEW

File: src/ai/backend/web/proxy.py [hunk 1]
```diff
@@ -206,11 +207,11 @@ async def web_handler(request: web.Request, *, is_anonymous=False) -> web.Stream
             if proxy_path == "pipeline":
                 aiohttp_session = request.cookies.get("AIOHTTP_SESSION")
                 if not (sso_token := request.headers.get("X-BackendAI-SSO")):
-                    jwt_secret = request.app["config"]["pipeline"]["jwt"]["secret"]
-                    now = datetime.now(tz=timezone(timedelta(hours=9)))
+                    jwt_secret = config["pipeline"]["jwt"]["secret"]
```
Comment (kyujin-cho) [inline src/ai/backend/web/proxy.py:211]: Please update config schema accordingly.

File: src/ai/backend/web/proxy.py [hunk 2]
```diff
@@ -206,11 +207,11 @@ async def web_handler(request: web.Request, *, is_anonymous=False) -> web.Stream
...
+                    now = datetime.now().astimezone()
                     payload = {
                         # Registered claims
-                        "exp": now + timedelta(hours=1),
+                        "exp": now + timedelta(seconds=config["session"]["max_age"]),
```
Comment (kyujin-cho) [inline src/ai/backend/web/proxy.py:215]: Same here.

**Review by kyujin-cho — APPROVED** [review]

**Review by kyujin-cho — CHANGES_REQUESTED** [review]
Oops I misclicked

File: src/ai/backend/web/proxy.py (hunk 2 above)
Comment (rapsealk) [reply src/ai/backend/web/proxy.py:215]: This one already has corresponding one: https://github.com/lablup/backend.ai/blob/9c512d54337aa53eb20f09c198bbffa179c71b57/src/ai/backend/web/config.py#L112-L122

**Pull request merged**